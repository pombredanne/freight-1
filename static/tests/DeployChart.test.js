import React from 'react';
import renderer from 'react-test-renderer';
import DeployChart from '../components/DeployChart.jsx';

test('DeployChart Snapshot', () => {
  const params = {
    "app": "freight",
    "env": "production",
    "number": "798"
  }
  const component = renderer.create(
    (<DeployChart params={params} />)
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
