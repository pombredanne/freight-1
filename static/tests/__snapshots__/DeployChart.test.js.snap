// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DeployChart Snapshot 1`] = `
<div
  className="section"
>
  <figure
    className=" barchart"
    height={undefined}
    width={undefined}
  >
    <span
      className="max-y"
    >
      10
    </span>
    <span
      className="min-y"
    >
      0
    </span>
  </figure>
</div>
`;
